WalletWasabi.Fluent/Rpc/WasabiJsonRpcService.cs

```diff
@@ -48,6 +51,19 @@ public object[] GetUnspentCoinList()
 			}).ToArray();
 		}
 
+		[JsonRpcMethod("listpockets")]
+		public object[] GetPockets()
+		{
+			var activeWallet = Guard.NotNull(nameof(ActiveWallet), ActiveWallet);
+			SourceList<PocketViewModel> pocketS = new();
+
+			AssertWalletIsLoaded();
+
+			pocketS.AddRange(activeWallet.Coins.GetPockets(0).Select(x => new PocketViewModel(x)));
```

Comment:
Here you are requesting those with anonimityset < 0 so, it will never return any result.